public/app/features/provisioning/HomePage.tsx

```diff
@@ -0,0 +1,169 @@
+import { useEffect, useState } from 'react';
+
+import { AppEvents } from '@grafana/data';
+import { getAppEvents } from '@grafana/runtime';
+import { Alert, ConfirmModal, Modal, Stack, Tab, TabContent, TabsBar } from '@grafana/ui';
+import { Page } from 'app/core/components/Page/Page';
+
+import { ExportToRepository } from './ExportToRepository';
+import { FilesView } from './FilesView';
+import { FolderRepositoryList } from './FolderRepositoryList';
+import GettingStarted from './GettingStarted/GettingStarted';
+import GettingStartedPage from './GettingStarted/GettingStartedPage';
+import { MigrateToRepository } from './MigrateToRepository';
+import { RepositoryActions } from './RepositoryActions';
+import { RepositoryOverview } from './RepositoryOverview';
+import { RepositoryResources } from './RepositoryResources';
+import { useDeletecollectionRepositoryMutation, useGetFrontendSettingsQuery } from './api';
+import { useRepositoryList } from './hooks';
+import { checkSyncSettings } from './utils';
+
+const appEvents = getAppEvents();
+
+enum TabSelection {
+  Overview = 'overview',
+  Resources = 'resources',
+  Files = 'files',
+  GettingStarted = 'getting-started',
+  Repositories = 'repositories',
+}
+
+const connectedTabInfo = [
+  { value: TabSelection.Overview, label: 'Overview', title: 'Repository overview' },
+  { value: TabSelection.Resources, label: 'Resources', title: 'Resources saved in grafana database' },
+  { value: TabSelection.Files, label: 'Files', title: 'The raw file list from the repository' },
+  { value: TabSelection.GettingStarted, label: 'Getting started', title: 'Getting started' },
+];
+
+const disconnectedTabInfo = [
+  { value: TabSelection.Repositories, label: 'Repositories', title: 'List of repositories' },
+  { value: TabSelection.GettingStarted, label: 'Getting started', title: 'Getting started' },
+];
+
+export default function HomePage() {
+  const [items, isLoading] = useRepositoryList({ watch: true });
+  const settings = useGetFrontendSettingsQuery();
```

Comment:
Settings data is just a trimmed down version of `useRepositoryList`, since you're already calling that here, you can use that data instead of fetching the settings as well.

Reply:
Weird, maybe it'd be added then? Otherwise it's weird to call almost the same endpoint twice.